modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/MarkedItems/IUserMarkedItemRepository.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Volo.Abp.Domain.Repositories;
+
+namespace Volo.CmsKit.MarkedItems;
+
+public interface IUserMarkedItemRepository : IBasicRepository<UserMarkedItem, Guid>
+{
+    Task<UserMarkedItem> FindAsync(
+        Guid userId,
+        [NotNull] string entityType,
+        [NotNull] string entityId,
+        CancellationToken cancellationToken = default
+    );
+
+    Task<List<UserMarkedItem>> GetListForUserAsync(
+        Guid userId,
+        [NotNull] string entityType,
+        CancellationToken cancellationToken = default
+    );
+
+    Task<IQueryable<UserMarkedItem>> GetQueryForUserAsync(
```

Comment:
Please do not expose `IQueryable` from repositories in the modules. Since I reviewed, this is not even used but tests, we can easily remove it

Reply:
![image](https://github.com/user-attachments/assets/3e95e119-c3bc-464b-b01d-7e5b31dbf99c)
I mentioned the need for the IQueryable method in the documentation... So, if it is better to remove it, we will remove it.

Reply:
Can you go with the same approach with tags?
https://github.com/abpframework/abp/blob/583f3c5b3cd80bb9820209f1b1bae611f840c011/modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Tags/IEntityTagRepository.cs#L25